modules/cert-manager-acme-challenges-types.adoc

```diff
@@ -12,6 +12,13 @@ HTTP-01:: With the HTTP-01 challenge type, you provide a computed key at an HTTP
 +
 For more information, see link:https://cert-manager.io/docs/configuration/acme/http01/[HTTP01] in the upstream cert-manager documentation.
 
+[NOTE]
+====
+HTTP-01 requires _Let's Encrypt_ servers to access the cluster's route. Note that using internal or private clusters behind the proxy to access, would fail the HTTP-01 validation checks required to issue a certificate.
```

Comment:
"Let's Encrypt" is sometimes italicized and sometimes not within the docs set. You're just replicating what's already in this assembly, anyway, but I believe ISG guidance would be to not highlight the name because it is not difficult to distinguish from surrounding text.

Reply:
I'm not sure if the subjects and objects are clear enough here. 

```suggestion
HTTP-01 requires that _Let's Encrypt_ servers can access the route of the cluster. If an internal or private cluster is behind a proxy, the HTTP-01 validations for certificate issuance fail.
```

Or something like that.

Reply:
Ack.